src/workflows-app/SubmissionDetails.test.js

```diff
@@ -690,4 +782,349 @@ describe('Submission Details page', () => {
+
+      const row3cells = within(rows[3]).getAllByRole('cell');
+      expect(row3cells.length).toBe(2);
+      expect(row3cells[0]).toHaveTextContent('docker');
+      expect(row3cells[1]).toHaveTextContent('quay.io/broadinstitute/ncbi-tools:2.10.7.10');
+    }
+  });
+
+  it('should open the task data modal when Outputs button is clicked', async () => {
+    // Arrange
+    const user = userEvent.setup();
+    const getRuns = jest.fn(() => Promise.resolve(mockRunsData));
+    const getRunsSets = jest.fn(() => Promise.resolve(runSetData));
+    const getMethods = jest.fn(() => Promise.resolve(methodData));
+    const mockLeoResponse = jest.fn(() => Promise.resolve(mockAzureApps));
+    Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runs: {
+            get: getRuns,
+          },
+          runSets: {
+            get: getRunsSets,
+          },
+          methods: {
+            getById: getMethods,
+          },
+        },
+        Apps: {
+          listAppsV2: mockLeoResponse,
+        },
+        CromwellApp: {
+          workflows: () => {
+            return {
+              metadata: jest.fn(() => {
+                return Promise.resolve(runDetailsMetadata);
+              }),
+            };
+          },
+        },
+        AzureStorage: {
+          blobByUri: jest.fn(() => ({
+            getMetadataAndTextContent: () =>
+              Promise.resolve({
+                uri: 'https://someBlobFilePath.blob.core.windows.net/cromwell/user-outputs/outputFile.txt',
+                sasToken: '1234-this-is-a-mock-sas-token-5678',
+                fileName: 'outputFile.txt',
+                name: 'outputFile.txt',
+                lastModified: 'Mon, 22 May 2023 17:12:58 GMT',
+                size: '324',
+                contentType: 'text/plain',
+                textContent: 'this is the text of a mock file',
+                azureSasStorageUrl: 'https://someBlobFilePath.blob.core.windows.net/cromwell/user-outputs/outputFile.txt',
+              }),
+          })),
+          details: jest.fn(() => {
+            return Promise.resolve({ sas: { token: '1234-this-is-a-mock-sas-token-5678' } });
+          }),
+        },
+        Metrics: {
+          captureEvent,
+        },
+      };
+    });
+
+    // Act
+    await act(async () => {
+      render(
+        h(BaseSubmissionDetails, {
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+          submissionId,
+          workflowId: '00001111-2222-3333-aaaa-bbbbccccdddd',
+          uri: 'https://coaexternalstorage.blob.core.windows.net/cromwell/user-inputs/inputFile.txt',
+        })
+      );
+    });
+
+    const outputsButtons = screen.getAllByRole('button', { name: 'Outputs' });
+    expect(outputsButtons !== undefined);
+
+    for (const outputsButton of outputsButtons) {
+      // Act
+      await user.click(outputsButton);
+      screen.logTestingPlaygroundURL();
```

Comment:
Looks like this is left over from development. Also, we have tests configured to fail on any output.

```suggestion
```